presto-mysql/src/main/java/com/facebook/presto/plugin/mysql/MySqlClient.java

```diff
@@ -115,32 +122,35 @@ protected SchemaTableName getSchemaTableName(ResultSet resultSet)
     @Override
     protected String toSqlType(Type type)
     {
-        if (Varchars.isVarcharType(type)) {
+        if (REAL.equals(type)) {
+            return "float";
+        }
+        if (TIME_WITH_TIME_ZONE.equals(type)) {
+            return "time";
+        }
+        if (TIMESTAMP.equals(type) || TIMESTAMP_WITH_TIME_ZONE.equals(type)) {
+            return "datetime";
+        }
+        if (VARBINARY.equals(type)) {
+            return "varbinary";
```

Comment:
We used to return `mediumblob`. I guess returning `varbinary` is conscious, but just in case...

Reply:
Good catch! This change was not intentional. Fixed